src/main/java/com/thealgorithms/backtracking/ArrayCombination.java

```diff
@@ -10,23 +10,35 @@
 public final class ArrayCombination {
     private ArrayCombination() {
     }
-    private static int length;
 
     /**
-     * Find all combinations of 1..n by creating an array and using backtracking in Combination.java
-     * @param n max value of the array.
-     * @param k length of combination
-     * @return a list of all combinations of length k. If k == 0, return null.
+     * Find all combinations of 1..n using backtracking.
```

Comment:
Wat do you think about
```suggestion
     * Find all combinations of 0..n-1 using backtracking.
```
Clearly this will require changes in the code and tests, but in my opinion it will be much more useful.